src/webhook_service/list_webhooks/lambda_function.py

```diff
@@ -0,0 +1,108 @@
+import json
+import logging
+import os
+from decimal import Decimal
+from typing import Dict
+
+import boto3
+from boto3.dynamodb.conditions import Key
+from get_total_count import get_total_count
+
+dynamodb = boto3.resource("dynamodb")
+table = dynamodb.Table(os.getenv("DYNAMODB_TABLE"))
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            # Convert Decimal to float or int, depending on your needs
+            return float(obj)
+        return super(DecimalEncoder, self).default(obj)
```

Comment:
Parameter 'obj' has been changed to 'o' to prevent overriding.
```
class DecimalEncoder(json.JSONEncoder):
    def default(self, o):
        if isinstance(o, Decimal):
            # Convert Decimal to float or int, depending on your needs
            return float(o)
        return super(DecimalEncoder, self).default(o)
```
Take a look at the [lambda_function](https://github.com/aws-educate-tw/aws-educate-tpet-backend/blob/SCRUM-157-list-webhooks/src/webhook_service/list_webhooks/lambda_function.py)